http/server.go

```diff
@@ -399,15 +352,79 @@ func (s *Server) handleExec(ctx *fasthttp.RequestCtx) {
 	s.replyOK(ctx)
 }
 
+func (s *Server) handleSimpleJsonACK(ctx *fasthttp.RequestCtx) {
+	// TODO: consider adding validation
+	s.replyOK(ctx)
+}
+
+func (s *Server) handleSimpleJsonQuery(ctx *fasthttp.RequestCtx) {
+	req, err := simplejson.SimpleJsonRequestFactory("query", ctx.PostBody())
+	if err != nil {
+		s.logger.ErrorWith("can't initialize simplejson query request", "error", err)
+		ctx.Error(fmt.Sprintf("bad request - %s", err), http.StatusBadRequest)
+		return
+	}
+	readRequest := req.GetReadRequest()
+	factory := backends.GetFactory(readRequest.Backend)
+	var backendConfig *frames.BackendConfig
```

Comment:
Use `s.api.Read` instead of manually creating a backend here.

Reply:
I had a discussion with @yaronha on this and he suggested using backend.Read instead of api.Read to avoid using a channel

Reply:
Why should we avoid using a channel?

Reply:
@tebeka the channel was from the http to the API (since we used streaming) AFAIK, here we dont have a front-end chan and no need for one

Reply:
After talking to @yaronha a bit more, it's better to use the API and not create backend every time, here's a code skeleton
```go
	ch := make(chan frames.Frame)
	var apiError error
	go func() {
		defer close(ch)
		apiError = s.api.Read(request, ch)
		if apiError != nil {
			s.logger.ErrorWith("error reading", "error", apiError)
		}
	}()

	for frame := range ch {
		// Do stuff here
	}

	if apiError != nil {
		s.writeError(enc, apiError)
	}
```